src/NetAnalyzers/UnitTests/Microsoft.CodeQuality.Analyzers/QualityGuidelines/AssigningSymbolAndItsMemberInSameStatementTests.Fixer.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Xunit;
+using VerifyCS = Test.Utilities.CSharpCodeFixVerifier<
+    Microsoft.CodeQuality.Analyzers.QualityGuidelines.AssigningSymbolAndItsMemberInSameStatement,
+    Microsoft.CodeQuality.Analyzers.QualityGuidelines.AssigningSymbolAndItsMemberInSameStatementFixer>;
+
+namespace Microsoft.CodeAnalysis.NetAnalyzers.UnitTests.Microsoft.CodeQuality.Analyzers.QualityGuidelines
+{
+    public class AssigningSymbolAndItsMemberInSameStatementFixerTests
```

Comment:
💡 Don't separate the analyzer and code fix tests. Rather, update the tests in-place to call `VerifyCodeFixAsync` instead of `VerifyAnalyzerAsync`.

* If no code fix is offered, use `VerifyCodeFixAsync(code, code)`. This validates that no code fix is offered for the scenario.
* If the code fix is offered, use `VerifyCodeFixAsync(code, fixedCode)`.

Reply:
Thanks. I'll update that soon.

Reply:
@sharwell FYI, there are some splitted tests in that same folder.